### ISSUE TITLE
Self-update when provisionning

### ISSUE THREAD

**Issue by benji07** [Elao/ansible-role-composer#1]
May be we could launch a composer self-update when we first launch our VM

**Comment by nervo** [comment]
From my point of view, this must be a manual action, like update other things on the vm.
The fact is, if we keep up-to-date vagrant boxes, we should'nt have issues with too old composer versions.
Agree ?

**Comment by nervo** [comment]
(and yes, the last box version is a little bit old, but i was waiting some important steps, ilke php 5.6, gulp 4, and other things i don't even remember :))

**Comment by benji07** [comment]
yes, but for old project we can't do that (what if we can't use php 5.6 or gulp 4)

**Comment by nervo** [comment]
php and gulp versions are always versionned by projects (have a look on ansible files)

**Comment by benji07** [comment]
#2
